psci/Parser.hs

```diff
@@ -86,3 +85,6 @@ parseCommand cmdString =
     where
     manyDecls :: P.TokenParser [P.Declaration]
     manyDecls = C.mark (many1 (C.same *> P.parseDeclaration))
```

Comment:
And I think we should potentially allow foreign import declarations, too. This currently works:

```
> let foreign import foo "var foo = 1 + 2" :: Number
> foo
3
```